tests/app/conftest.py

```diff
@@ -18,7 +18,11 @@ def app(mocker):
     mocker.patch("app.setup.datastore.Client", MockDatastore)
     mocker.patch("app.setup.redis.Redis", fakeredis.FakeStrictRedis)
     the_app = create_app(setting_overrides=setting_overrides)
-    return the_app
+    the_app.config["SERVER_NAME"] = "test.localdomain"
+    app_context = the_app.app_context()
+    app_context.push()
+    yield the_app
+    app_context.pop()
```

Comment:
Not sure about this, I might have to do some reading up

Reply:
You mean the pop/push? I had to implement it in the fixture for some tests to work properly. Honestly I don't know the specifics on the usage and why it's implemented in the first place, if not then some tests will need refactoring.

Reply:
No worries, I was doing the review and it caught my eye and I didn't know much about it